core/storage/feedback/gae_models.py

```diff
@@ -200,8 +200,9 @@ def generate_new_thread_id(cls, entity_type, entity_id):
         for _ in python_utils.RANGE(_MAX_RETRIES):
             thread_id = (
                 entity_type + '.' + entity_id + '.' +
```

Comment:
Could we use interpolation instead to construct this?

Reply:
Yes. Done.